utils/frame/benchmarking-cli/src/writer.rs

```diff
@@ -200,20 +239,20 @@ pub fn write_results(batches: &[BenchmarkBatch]) -> Result<(), std::io::Error> {
 		})?;
 
 		if !reads.base.is_zero() {
-			write!(file, "\t\t\t.saturating_add(DbWeight::get().reads({} as Weight))\n", reads.base)?;
+			write!(file, "\t\t\t.saturating_add(T::DbWeight::get().reads({} as Weight))\n", reads.base)?;
```

Comment:
Can we also introduce a new flag `--space` to use Space instead of Tab here? Substrate uses tab, but `cargo format` uses space by default.

Reply:
yes, I can add this 👍